src/Math/Matrix.h

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <cmath>
```

Comment:
Why are standard C/C++ libraries included here? If I remember correctly we decided to try to stay away from them for the Teensy even though they are supported, but iostream definitely won't work on a Teensy.

Reply:
Left over from porting from KF repo